Documentation/quickstart.md

```diff
@@ -29,6 +29,7 @@ If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, mak
 If you're feeling lucky, a simple Rook cluster can be created with the following kubectl commands. For the more detailed install, skip to the next section to [deploy the Rook operator](#deploy-the-rook-operator).
 ```
 cd cluster/examples/kubernetes
+kubectl create -f rook-crds.yaml
```

Comment:
can we avoid adding another file, and just create the CRDs inside rook-crds.yaml?